### DISCUSSION TITLE
Is there any support for drag-drop-reorder list items?

### DISCUSSION THREAD

**Discussion by ClaudiuHBann** [amwx/FluentAvalonia#506] [Q&A] [answered]
I have a list and i want to reorder items by drag and drop between the list items, is there any built in support for this in AvaloniaUI?
You know any library for this? I am using an ItemsRepeater BTW

**Comment by timunie** [comment, marked as answer]
Check out Avalonia.Behaviors. if there is no Behavior suits your needs, it's easy to copy one and modify it to your needs.

**Reply by ClaudiuHBann** [reply]
I found a simple example of what I want to achieve.
It's easy to copy an example but not to modify :))
I should add an item overlay of the item that is being moved, a line between the items that the dragged item is going to be dropped, if you have a lot of items in the list and you want to move the last item before the first item you should drag your item to the top of the list and it should scroll up, etc...
